oauthproxy.go

```diff
@@ -884,7 +884,7 @@ func (p *OAuthProxy) getAuthenticatedSession(rw http.ResponseWriter, req *http.R
 	}
 
 	if session != nil && session.Email != "" {
-		if !p.Validator(session.Email) || !p.provider.ValidateGroup(session.Email) {
+		if !p.Validator(session.Email) || !p.provider.ValidateGroup(session) {
```

Comment:
This doesn't work for me. The session object here does not have an ID token. Once I removed the ValidateGroup call it worked, but I guess that is not a good solution.

Reply:
I fixed it. The reason was that the ID token is only added to the cookie if certain options are used. I fixed it by setting a cookie-refresh duration.